scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -1,35 +1,35 @@
+title: "SARS-CoV-2 variant report"                                                                                                  
 date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
-theme: darkly
 params:  
-  vep_txt_file: ''
-  snv_csv_file: ''
-  location_sigmuts: ''
-  sample_dir: ''
-  sample_name: ''
-  outFile: ''
+  @PARAMS_SAMPLE_NAME@
```

Comment:
`@...@` is used for placeholders in `.in` files, but this is not a `.in` file, so this placeholder will never be replaced with a concrete value.  What is the intention here?

Reply:
That placeholder is something I used for the crispr-dart pipeline. The idea is a copy of this report script is created for each sample, the parameter (in this case the name of the sample) for the sample report is replaced for the sample using a `grep` which looks for this place holder. See here: https://github.com/BIMSBbioinfo/pigx_sarscov2_ww/blob/0baa72fe3ee659c2c23b3486d9bb807989fc05d9/scripts/generateSiteFiles.R#L52-L54

Reply:
Ah, good.  Same convention but runtime replacement instead of configure-time replacement.  Got it!  Thanks!